bigbluebutton-web/grails-app/conf/bigbluebutton.properties

```diff
@@ -178,11 +178,6 @@ defaultGuestPolicy=ALWAYS_ACCEPT
 # Enables or disables authenticated guest
 authenticatedGuest=true
 
-#---------------------------------------------------
-# Default Meeting Layout
-# Valid values are CUSTOM_LAYOUT, SMART_LAYOUT, PRESENTATION_FOCUS, VIDEO_FOCUS
-defaultMeetingLayout=SMART_LAYOUT
```

Comment:
I am not sure if removing this is enough - see https://github.com/bigbluebutton/bigbluebutton/pull/12789/files

Reply:
It's enough because I'm ignoring the value coming from bbb-web, but we should also remove all the now unused code

Reply:
Also related https://github.com/bigbluebutton/bigbluebutton.github.io/pull/295/files